### DISCUSSION TITLE
Add comments to selected content

### DISCUSSION THREAD

**Discussion by shenxiangzhuang** [squidfunk/mkdocs-material#7131] [Ideas]
Is it possible to add comments to selected content(like [lark-docs](https://www.larksuite.com/hc/en-US/articles/680083197034-add-comments-in-lark-docs) and [notion](https://www.notion.so/help/comments-mentions-and-reminders) did)?

It's an awesome feature I think, which can help a lot for question discussion.

**Comment by alexvoss** [comment]
I'll assume first that what you are looking for is a comment system for the users of your site? You can [integrate a discussion system like Giscus](https://squidfunk.github.io/mkdocs-material/setup/adding-a-comment-system#adding-a-comment-system). This allows you to attach a discussion to each page, which will usually sit at the bottom of the page. Note that it is somewhat out of scope for a static site generator to provide commenting functionality for the finished site as this inherently requires some infrastructure.

If you want to simply leave notes for your readers, you might be able to utilize [tooltips](https://squidfunk.github.io/mkdocs-material/reference/tooltips#tooltips) or [footnotes](https://squidfunk.github.io/mkdocs-material/reference/footnotes/). In the Insiders Edition you can even combine the two.

**Reply by shenxiangzhuang** [reply]
Thanks a lot for your timely reply! I agree with that it maybe out of scope for this project. I'll checkout your suggestions later.

**Comment by squidfunk** [comment]
It's rather an interactive feature that is beyond static sites, but if someday somebody builds a WYSIWYG editor for Material for MkDocs, this would definitely be a useful feature. Regardless, out of scope for now 😅
